src/com/qmetry/qaf/automation/elasticsearch/ElasticSerachService.java

```diff
@@ -49,12 +49,12 @@
  *
  */
 public class ElasticSerachService {
-	private static final ElasticSerachService SERVICE = new ElasticSerachService();
```

Comment:
Looks like committed by mistake.

Reply:
Actually intensely I have changed line order, because when it is going to create a new instance of ElasticSerachService, it uses INDEX_NAME variable which have been not initialised yet. So first it was creating index with 'null' name.

I have debug it and found that root cause only, still needs your input.